src/css/components/_modal-dialog.scss

```diff
@@ -0,0 +1,33 @@
+.video-js .vjs-modal-dialog {
+  @extend %fill-parent;
+  @include linear-gradient(180deg, rgba(0, 0, 0, 0.8), rgba(255, 255, 255, 0));
+}
+
+.vjs-modal-dialog .vjs-modal-dialog-content {
+  @extend %fill-parent;
+  z-index: 1;
+}
+
+// Ultimately, this would be moved to its own component partial, but
+// for now it's unclear what styles would be generic and which specific
+// to this context!
+.vjs-modal-dialog .vjs-close {
```

Comment:
Hmm. Looks like the cancel icon is not released in videojs/font yet. It is [available in master](https://github.com/videojs/font/commit/182aa4c2ee46cca7ca00155c4cdbd89acbad5a65#diff-3709d0d9b83e562f3bfdb9483dd92c9f), though.